sktm/patchwork.py

```diff
@@ -237,24 +237,25 @@ def stringify(value):
     return str(value)
 
 
-# Internal RH PatchWork adds a magic API version with each call
-# this class just magically adds/removes it
 class RpcWrapper(object):
+    """
+    Class for removal of magic API version which is added by RH Patchwork fork.
+    """
     def __init__(self, real_rpc):
         self.rpc = real_rpc
-        # patchwork api coded to
+        # RH-Patchwork API version
         self.version = 1010
 
     def _wrap_call(self, rpc, name):
-        # Wrap a RPC call, adding the expected version number as argument
+        """Wrap a RPC call, adding the expected version number as argument."""
         function = getattr(rpc, name)
 
         def wrapper(*args, **kwargs):
             return function(self.version, *args, **kwargs)
         return wrapper
 
     def _return_check(self, returned):
-        # Returns just the real return value, without the version info.
+        """Returns the real return value without the version info."""
```

Comment:
Do you think you can go all the way here, and add argument and return value documentation?

And maybe even do the one above as well?

Not required for my ACK on this.

Reply:
@spbnick could I ask you questions? What does ‘ your ACK’ mean?

Reply:
"ACK" is short for "acknowledgment". The abbreviation comes, from networking terms, probably. Here, it means "approve the PR" :)